feature/bgp/prefixlimit/otg_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go

```diff
@@ -194,58 +188,67 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) *config {
 	dstBgp6Peer := dstBgp.Ipv6Interfaces().Add().SetIpv6Name(dstIpv6.Name()).Peers().Add().SetName(ateDst.Name + ".BGP6.peer")
 	dstBgp6Peer.SetPeerAddress(dstIpv6.Gateway()).SetAsNumber(ateAS).SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
 
-	dstBgp4PeerRoutes := dstBgp4Peer.V4Routes().Add().SetName(ateDst.Name + ".BGP4.peer" + ".RR4")
-	dstBgp4PeerRoutes.SetNextHopIpv4Address(dstIpv4.Address()).
+	configureBGPv4Routes(dstBgp4Peer, dstIpv4.Address(), "r4UnderLimit", Routesv4NetUnderLimit, prefixLimit-1)
```

Comment:
Let's make the names of the route objects global constants and then refer to them in the testcases directly. (As in test cases include the explicit route object names instead of deriving from the test case name.) Then the advertise/withdraw functions can take in a list of route object names instead.

Reply:
updated